indra/newview/scripts/lua/test_luafloater_demo.lua

```diff
@@ -0,0 +1,69 @@
+XML_FILE_PATH = "luafloater_demo.xml"
+
+leap = require 'leap'
+coro = require 'coro'
+
+--event pump for sending actions to the floater
+COMMAND_PUMP_NAME = ""
+--table of floater UI events
+e={}
+coro.launch(function ()
+  e = leap.request("LLFloaterReg", {op="getFloaterEvents"})["events"]
+  leap.done()
+end)
+leap.process()
```

Comment:
Heh! Going forward, it should also work to just execute line 11 from the main thread. But I don't have that finished yet.

Reply:
Looking forward. As you suggested, I'll change `leap.send` (when opening a floater) to `leap.request` after that.

Reply:
Actually, now that `postBuild()` echoes the incoming `"reqid"`, that should work now ... from within a coroutine, until I finish the main-thread support. I guess that's what you meant.